packages/cozy-client/src/StackLink.js

```diff
@@ -1,10 +1,24 @@
-import { MutationTypes } from './queries/dsl'
+import { MutationTypes, QueryDefinition } from './queries/dsl'
 import CozyLink from './CozyLink'
-import { CozyClientDocument, CouchDBBulkResult } from './types'
+import { CozyClientDocument, CouchDBBulkResult, ClientResponse } from './types'
 import { DOCTYPE_FILES } from './const'
 import { BulkEditError } from './errors'
 import zipWith from 'lodash/zipWith'
-
+/**
+ *
+ * To know if cozy-client should use Document.find()
+ * or Document.all()
+ * Similar to what is done in CozyPouchLink executeQuery()
+ *
+ * @param {QueryDefinition} queryDefinition - QueryDefinition to check
+ * @returns {boolean} If has find options
+ *
+ */
+const hasFindOptions = queryDefinition => {
+  const { selector, partialFilter, sort, fields } = queryDefinition
+  if (selector || partialFilter || sort || fields) return true
+  return false
```

Comment:
don't you prefer `return selector || partialFilter || sort || fields` ?

Reply:
I do not have any pref. At first, this method was more complicated than just returning the fields. Since I wrote it that way, I will commit that way. Maybe next time you'll change it :p